### PR TITLE
[ticket/13863] Add topiclist_row_append/prepend to mcp_forum.html

### PR DESCRIPTION
**Pull request by imkingdavid**
These events are already present in viewforum_body.html and search_results.html, just adding them to mcp_forum.html

https://tracker.phpbb.com/browse/PHPBB3-13863

### CODE REVIEW

File: phpBB/docs/events.md [hunk 1]
```diff
@@ -1184,8 +1184,10 @@ topiclist_row_prepend
 * Locations:
     + styles/prosilver/template/search_results.html
     + styles/prosilver/template/viewforum_body.html
+    + styles/prosilver/template/mcp_forum.html
```
Comment (nickvergessen) [inline phpBB/docs/events.md]: I think we should add a `* Changed: 3.1.5-RC1` ?

Comment (marc1706) [inline phpBB/docs/events.md]: Should add ```* Changed: 3.1.5-RC1 Added event to mcp_forum.html```

Comment (nickvergessen) [inline phpBB/docs/events.md]: @imkingdavid can you add that?

**Comment by marc1706** [comment]
Replaced by #3797

**Pull request closed without merging**